src/site/content/en/learn/css/pseudo-classes/pseudo-classes.assess.yml

```diff
@@ -0,0 +1,62 @@
+setLeader: Test your knowledge of pseudo classes
+height: unset
+tabLabel: question
+questions:
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "0"
+    stem: |
+      Pseudo-classes select elements based on internal browser state, while
+      Pseudo-elements select browser owned parts of elements?
+    options:
+      - content: "True"
+        rationale: "Watch for the use of a single or double `:` as a key
+        distinguishing character in the selector"
+      - content: "False"
+        rationale: "Pseudo-elements are for parts, Pseudo-classes are for
+        state."
+  - type: multiple-choice
+    cardinality: "1"
+    correctAnswers: "1"
+    stem: Like psuedo-elements, all CSS styles are available for use?
```

Comment:
```suggestion
    stem: In psuedo-elements, all CSS styles are available for use.
```

Reply:
This changes the focus to elements though, and the question intends to quiz on classes. The idea was to help them remember the limits of pseudo element styling and consider that pseudo classes will have the same constraints. Thoughts on how we can find a middle ground?

Reply:
Maybe don't mention pseudo-elements at all? 

You can style a pseudo-class using any available CSS properties.

Reply:
yeah, i'm with you now. this question isnt focused. i'm gonna remove it and maybe come up with a new one!